### PR TITLE
cleanup container files

### PR DESCRIPTION
**Pull request by engelmi**
This PR does
- prune container files for integration tests
- added base images to quay.io for 
   - building hirte (reduces execution time of pipeline)
   - running integration tests for hirte

Note: Currenly, the images are manually pushed to quay.io/hirte. In a future PR this should be automated. Till then, for updating the images please reach out to @engelmi.

### CODE REVIEW

File: tests/containers/integration-test-base [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+RUN dnf install \
+        systemd-devel \
+        systemd \
+        selinux-policy \
+    -y
+
+RUN dnf -y clean all
+
+WORKDIR /hirte
```
Comment (mwperina) [inline tests/containers/integration-test-base]: Do we need this directory for anything? Isn't it only remain from the time before conversion to tmt powered tests?

Comment (engelmi) [reply tests/containers/integration-test-base]: Probably not. I am also thinking about using a non-root user to limit privileges.

File: tests/containers/build-base [hunk 2]
```diff
@@ -0,0 +1,29 @@
+FROM quay.io/centos/centos:stream9
+
+RUN dnf install -y dnf-plugin-config-manager
+# CRB is required for meson
+RUN dnf config-manager -y --set-enabled crb
+# EPEL is required for python3-flake8 and codespell
+RUN dnf install -y epel-release
+
+RUN dnf install \
```
Comment (mwperina) [inline tests/containers/build-base:9]: Could you please sort packages by their names to ease maintenance?

File: tests/containers/integration-test-base [hunk 3]
```diff
@@ -0,0 +1,12 @@
+FROM quay.io/centos/centos:stream9
+
+RUN dnf install -y dnf-plugin-config-manager
+
+RUN dnf install \
+        systemd-devel \
+        systemd \
+        selinux-policy \
+        policycoreutils-python-utils \
```
Comment (mwperina) [inline tests/containers/integration-test-base:6]: Same as above

File: tests/containers/build-base (hunk 2 above)
Comment (engelmi) [reply tests/containers/build-base:9]: Done.

File: tests/containers/integration-test-base (hunk 3 above)
Comment (engelmi) [reply tests/containers/integration-test-base:6]: Done.

**Review by mwperina — APPROVED** [review]
LGTM, the CI for container updates can be added in a separate PR

**Pull request merged**